x-pack/legacy/plugins/lens/public/_config_panel.scss

```diff
@@ -19,3 +19,13 @@
   box-shadow: none !important;
   border: 1px dashed currentColor;
 }
+
+.lnsConfigPanel__dimension {
```

Comment:
This selector with styles seems to have been added, but I don't see the className being applied to anything in this PR?

Reply:
It's not added, it's moved.

Reply:
The CSS moved or the component with the class did? Either way, the old CSS needs to be removed then.

Reply:
Yes, it's removed.

Reply:
Ahh sorry, freakin github was hiding the "large diff" file this class existed in so doing a <kbd>ctrl+f</kbd> resulted in only this one result. I see the usage now.